src/test/java/com/spring/boot/project/demo/unit/ParserReviewCsvTest.java

```diff
@@ -0,0 +1,67 @@
+package com.spring.boot.project.demo.unit;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import com.spring.boot.project.demo.model.AmazonUser;
+import com.spring.boot.project.demo.model.Product;
+import com.spring.boot.project.demo.model.Review;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParserReviewCsvTest {
+    private final FileReader fileReader = new FileUtil();
+    private final Parser parsedReviewParser = new ParserReviewCsv();
+    private final static String REVIEW_TEST_PATH =
+            "src/test/resources/tests/utilTest/reviews_test.csv";
```

Comment:
по ідеї повинно працювати і так:
```suggestion
            "tests/utilTest/reviews_test.csv";
```
до речі неймінг пакетів в java завжди з маленької літери)))

Reply:
Ну так він і є з маленької. Ти напевно хотіла сказати, що повинене містити лише малі лірети, і якщо це так як робити відокремлення слів між собою?

Reply:
в такому випадку потрібно або замінити на назву з одним словом, або ж написати все разом